test/text-equals.test.js

```diff
@@ -0,0 +1,119 @@
+test('Using "eq" returns true if the left value equals the right value ', async () => {
+  const result = await $app['equalsText 1.0']({ leftValue: 'Lorum Ipsum', rightValue: 'Lorum Ipsum', comparator: 'eq', toLower: false });
```

Comment:
Don't declare the same parameter everytime. Declare shared variable if values are reused